app/main.py

```diff
@@ -1,3 +1,55 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, second: Distance | int | float) -> any:
+        if isinstance(second, int) or isinstance(second, float):
+            return Distance(self.km + second)
+        return Distance(self.km + second.km)
+
+    def __iadd__(self, second: Distance | int | float) -> any:
+        if isinstance(second, int) or isinstance(second, float):
+            self.km += second
+            return self
+        self.km += second.km
+        return self
+
+    def __mul__(self, second: int | float) -> any:
+        return Distance(self.km * second)
+
+    def __truediv__(self, second: int | float) -> any:
+        return Distance(round(self.km / second, 2))
+
+    def __eq__(self, second: Distance | int | float) -> any:
+        if isinstance(second, int) or isinstance(second, float):
+            return self.km == second
+        return self.km == second.km
+
+    def __lt__(self, second: Distance | int | float) -> any:
```

Comment:
Indicate that you return `bool`, not `any`. Fix it in all places where applicable.